ansible/roles/aws_ecs/files/etc/init/ecs_agent.conf

```diff
@@ -8,7 +8,7 @@ respawn limit 100 1
 
 # Needed to find /root/.dockercfg
 env HOME="/root"
-env IMAGE=amazon/amazon-ecs-agent@sha256:54e98d053d0fd6735732e2c17054ab5d78b40be9d20a628fce993dc789240399 # v1.2.1 tag
+env IMAGE=amazon/amazon-ecs-agent@sha256:72c02d9a96340e4381c70b4355289e204b36e7e667352eed8df4d0e2f2d308a4 # v1.3.1 tag
```

Comment:
To get the sha you just do a pull on the image you want - the second to last line is the sha:

```
# docker pull amazon/amazon-ecs-agent:v1.3.1
v1.3.1: Pulling from amazon/amazon-ecs-agent
24f7ccba562b: Already exists
3c0df86ee4a3: Already exists
715b22385e84: Already exists
07d38398907c: Already exists
eab832a891a2: Already exists
Digest: sha256:72c02d9a96340e4381c70b4355289e204b36e7e667352eed8df4d0e2f2d308a4
Status: Downloaded newer image for amazon/amazon-ecs-agent:v1.3.1
```